src/GoogleAdWordsEventForwarder.js

```diff
@@ -166,6 +210,36 @@
             }
         }
 
+        function getUserConsentState() {
+            var userConsentState = null;
+
+            if (window.mParticle && window.mParticle.Identity && window.mParticle.Identity.getCurrentUser) {
+                var currentUser = window.mParticle.Identity.getCurrentUser();
+
+                if (!currentUser) {
+                    return null;
+                }
+
+                var consentState =
+                    window.mParticle.Identity.getCurrentUser().getConsentState();
+
+                if (consentState && consentState.getGDPRConsentState) {
+                    userConsentState = consentState.getGDPRConsentState();
+                }
+            }
+
+            return userConsentState;
+        }
+
+        function getEventConsentState(event) {
+            var eventConsentState = null;
+
+            if (event && event.ConsentState) {
+                eventConsentState = event.ConsentState.getGDPRConsentState();
+            }
+
+            return eventConsentState;
+        }
```

Comment:
I like what you're doing [here ](https://github.com/mparticle-integrations/mparticle-javascript-integration-google-analytics-4/pull/56/files#diff-f4fc97a32eb0f54aa94802d6fb59ef81c28b4e5cc2d050c0652e0940c0d99570R48-R52) in terms of just passing `eventConsentState` into the function.  I think it's cleaner.